New pages (#238)
NilPointer-Software/bdfd-wiki: src/bdscript/resetChannelVar.md

# $resetChannelVar

> Resets a channel variable back to it's default value *(the one inputted in the app)* **for every server**.

Reviewer: ```suggestion
Resets a channel variable back to it's default value *(the one provided in the app)* for every server.
```

> Resets a channel variable back to it's default value *(the one inputted in the app)* **for every server**.

Reviewer 2: ```suggestion
Resets a channel variable back to its default value *(the one provided in the app)* **for every channel in every server**.
```

> 

Reviewer 3: ```suggestion
```

> > 🧙‍♂️ Use this function wisely!

Reviewer 3: ```suggestion
```

> ## Usage

Reviewer: ```suggestion
## Syntax
```

> ## Usage

Reviewer 2: ```suggestion
## Syntax
```

```

> $resetChannelVar[variableName]

Reviewer: ```suggestion
$resetChannelVar[Name]
```

> $resetChannelVar[variableName]

Reviewer 2: ```suggestion
$resetChannelVar[Variable Name]
```

```

> ### Breakdown

Reviewer: ```suggestion
### Parameters
```

> - `variableName` - The name of the variable to reset.

Reviewer: ```suggestion
- `Name`: The name of the variable to reset.
```

> ### Breakdown
> - `variableName` - The name of the variable to reset.

Reviewer 2: ```suggestion
### Parameters
- `Variable Name`: The name of the variable to reset.
```
